bookkeeper-server/conf/bk_server.conf

```diff
@@ -296,6 +296,10 @@ journalDirectory=/tmp/bk-txn
 # Should we remove pages from page cache after force write
 # journalRemoveFromPageCache=false
 
+# Should the data be fsynced on journal before acknowledgment
```

Comment:
we need a warning here

Reply:
sure. I would suggest also marking it as `@experimental`

Reply:
fyi, I created an issue #756 (we need some guidelines on providing guidelines for a feature introduced in a release).

Reply:
Sure, I'll add more explicit docs on the risks of enabling the flag.